viewer-prototype/src/node/trace-server-service.ts

```diff
@@ -0,0 +1,14 @@
+import { spawn, exec } from 'child_process';
+import { injectable } from 'inversify';
+import { TraceServerConfigService } from '../common/trace-server-config';
+@injectable()
+export class TraceServerServiceImpl implements TraceServerConfigService {
+
+    async startTraceServer(path: string | undefined, port: number | undefined): Promise<void> {
+        spawn(`..${path}`, ['-vmargs', `-Dtraceserver.port=${port}`]);
```

Comment:
Are the ".." before the path intentional? When I put the path in the the settings /usr/bin/tracecompass-server then it tries to execute "../usr/bin/tracecompass-server" which fails and cause weird behaviour when opening a trace (i.e. it hangs the tsp calls).

Reply:
Fixed.